timemachine/cpp/src/barostat.hpp

```diff
@@ -8,18 +8,6 @@
 
 namespace timemachine {
 
-//class Barostat {
-//
-//public:
-//
-//    virtual void inplace_move(
-//        double *d_x,
-//        double *d_box,
-//        const double lambda
-//    );
-//
-//};
-
```

Comment:
Didn't seem worth keeping this up to date now that we should be able to get signatures in the IDE via `custom_ops.pyi`